pkg/crd/testdata/oneof/types.go

```diff
@@ -67,6 +69,20 @@ type TypeWithMultipleExactOneofs struct {
 	D *string `json:"d,omitempty"`
 }
 
+// +kubebuilder:validation:AtMostOneOf=a;b
+// +kubebuilder:validation:ExactlyOneOf=c;d
+// +kubebuilder:validation:AtLeastOneOf=e;f
+type TypeWithAllOneofs struct {
```

Comment:
To match the others?
```suggestion
type TypeWithAllOneOfs struct {
```

Reply:
I don't want to increase the scope of this PR, given that has been an issue with other renames/formatting in previous PRs of mine. Let's keep this as-is since it has no functional effects